lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -582,11 +582,24 @@ module.exports = class MethodMappings {
 
           return value;
         },
-
+        
         async clearElementValue(webElementOrId) {
+         
           const element = this.getWebElement(webElementOrId);
           await element.clear();
-
+          try{
+          const value = await element.getProperty("value");
+          if(value.length !== 0){
+          const backArr = Array(value.length).fill(Key.BACK_SPACE);
+          await element.sendKeys(...backArr);
+          }
+          }
+          catch(err){
+            return{
+            error: err.name,
+            message: err.message,
+            }
```

Comment:
I think if there's some error in the `try` block, we should just let it fail silently, like it would have if there were no fallback. This is because, if any error is to occur here, it will happen with the `getProperty('value')` command, and because this command will run every time (even if the `clear()` command ran successfully or not), we shouldn't bother users with errors while running this.

By silently, I mean to have an empty `catch` clause.